src/e-start.js

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env node
+
+const childProcess = require('child_process');
+const path = require('path');
+const fs = require('fs');
+const semver = require('semver');
+const program = require('commander');
+
+const evmConfig = require('./evm-config');
+const { color, fatal } = require('./utils/logging');
+
+program
+  .allowUnknownOption()
+  .option(
+    '--v <version>',
+    'A specific version binary of Electron to run the specified app with. Must have set the binary directory in the current config.',
+  )
+  .option('--path <path>', 'Path to an Electron executable')
+  .on('--help', () => {
+    console.log('');
+    console.log('Examples:');
+    console.log('');
+    console.log('  $ e start .');
+    console.log('  $ e start /path/to/app');
+    console.log('  $ e start --v=1.2.3 /path/to/app');
+    console.log('  $ e start /path/to/app --js-flags');
+  })
+  .parse(process.argv);
+
+try {
+  const config = evmConfig.current();
+
+  let exec;
+  let args;
+  if (program.v) {
+    const binDir = config.electronBinaryDirectory;
+    if (!binDir || !fs.existsSync(binDir)) {
```

Comment:
```suggestion
    if (!binDir) {
```

This is one of those little things that adds up -- `!fs.existsSync(binDir)` gives false security here b/c it confirms there's something in that path, but doesn't confirm it's a directory -- so we don't actually know that binDir is valid when doing this validity test.

We could (a) leave it as a halfway test or (b) add try-catch-statSync(), which is sturdier but less concise, or (c) do nothing and let the `path.resolve()` test below handle it. IMO (c) is preferable since `path.resolve()` really is the true arbiter, so duplication is unnecessary.

Again, not a huge deal -- the code as-is would work. I've just learned to distrust half-tests as a red flag.